kjdoc/tour.md

```diff
@@ -1021,7 +1021,24 @@ KJ provides an advanced, cross-platform filesystem API in `kj/filesystem.h`. Fea
 * Symlinks, hard links, listing directories, recursive delete, recursive create parents, recursive copy directory, memory mapping, and unnamed temporary files are all exposed and easy to use.
 * Sparse files ("hole punching"), copy-on-write file cloning (`FICLONE`, `FICLONERANGE`), `sendfile()`-based copying, `renameat2()` atomic replacements, and more will automatically be used when available.
 
+```c++
+// NOTE: in most cases, you should construct the `Filesystem` at the entry point of your program,
+// then pass it in as a parameter wherever it's used. This allows test implementations or user-space
+// filesystems.
+kj::Own<kj::Filesystem> fs = kj::newDiskFilesystem();
+kj::Own<const kj::ReadableFile>> file = fs->getCurrent().openFile(kj::Path::parse("README.md"));
+kj::String text = file->readAllText();
+```
+
+For large files, consider using mmap when you want to read the full contents of a file, which uses
+on demand paging as contents are accessed rather than buffering the contents into memory first.
```

Comment:
```suggestion
For large files, consider using mmap when you want to read the full contents of a file, which uses on demand paging as contents are accessed rather than buffering the contents into memory first.
```

We actually don't wrap prose in this document.